dockerfile/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
bikeshedding again -- how about naming that file `make-ci.sh`? We may have other scripts in the future that this container can run.